js/mexc.js

```diff
@@ -39,6 +40,11 @@ module.exports = class mexc extends Exchange {
                 'fetchDeposits': true,
                 'fetchFundingRate': true,
                 'fetchFundingRateHistory': true,
+                'fetchFundingRates': false,
+                'fetchIndexOHLCV': false,
+                'fetchIsolatedPositions': false,
+                'fetchLeverage': false,
+                'fetchMarkOHLCV': false,
```

Comment:
Double check this one, it's needed for funding fee calculations, so it's kind of important

Reply:
Which one exactly?

Reply:
`fetchMarkOHLCV`

Reply:
It's not super critical, but it's needed for backtesting